admin/spec/back-end/controllers/restart.spec.js

```diff
@@ -1,11 +1,14 @@
 'use strict'
-/* global describe it expect beforeEach jasmine spyOn fail */
+/* global describe it expect beforeEach jasmine spyOn fail xit */
```

Comment:
Why are the tests added commented out with `xit`, is it because of the feature toggle? 🤔 If so, can that be mocked and the tests ran?

Reply:
Yes, these tests are part of the HPA code paths.  

We may need to run unit tests in legacy mode, and also in HPA mode?  If so we could feature toggle the tests.

Perhaps we need a new ticket though?

Reply:
New ticket sounds sensible 👍